include/net/fota_download.rst

```diff
@@ -1,22 +1,28 @@
 .. _lib_fota_download:
 
-FOTA Download
+FOTA download
 #############
 
-The firmware over-the-air (FOTA) download library provides functions to download a firmware file as an upgrade candidate to the secondary slot of MCUboot.
+The firmware over-the-air (FOTA) download library provides functions for downloading a firmware file as an upgrade candidate to the DFU target that is used in the :ref:`lib_dfu_target` library.
 
-This is done using the :ref:`lib_download_client` and ``flash_img`` libraries.
+The library uses the :ref:`lib_download_client` to download the image.
+Once the download has been started, all received data fragments are passed to the :ref:`lib_dfu_target` library.
+The :ref:`lib_dfu_target` library takes care of where the upgrade candidate is stored, depending on the image type that is being downloaded.
 
-Once the download has been started, all received data fragments are passed to the ``flash_img`` library, which stores them in the appropriate memory address for an firmware upgrade candidate.
+When the download client sends the event indicating that the download has completed, the received firmware is tagged as an upgrade candidate, and the download client is instructed to disconnect from the server.
+The library then sends an :cpp:enumerator:`FOTA_DOWNLOAD_EVT_FINISHED<fota_download::FOTA_DOWNLOAD_EVT_FINISHED>` callback event.
```

Comment:
Tiny comment - now it should be "sends a ...FOTA", not "an".

Reply:
Fixed.